lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -53,43 +53,43 @@ def store_column(column_title, format_string, new_data):
         store_column(
             f"counts (summed over {2 * sampling_width + 1} pixels)",
             "%d",
-            np.hstack([line.sample_from_image(sampling_width) for line in kymoline_group]),
+            np.hstack([track.sample_from_image(sampling_width) for track in kymotrack_group]),
         )
 
     data = np.vstack(data).T
     np.savetxt(filename, data, fmt=fmt, header=delimiter.join(header), delimiter=delimiter)
 
 
-def import_kymolinegroup_from_csv(filename, kymo, channel, delimiter=";"):
-    """Import kymolines from csv
+def import_kymotrackgroup_from_csv(filename, kymo, channel, delimiter=";"):
+    """Import a KymoTrackGroup from a csv file.
 
     Parameters
     ----------
     filename : str
         filename to import from.
     kymo : Kymo
-        kymograph instance that these lines were tracked from.
+        kymograph instance that was tracked.
     channel : str
-        color channel that these lines were tracked from.
+        color channel that these coordinates were tracked from.
```

Comment:
Which coordinates? Probably:
```suggestion
        Color channel that the coordinates were tracked from.
```
or
```suggestion
        Color channel that was used for tracking.
```